Python/Iniciante/1049.py

```diff
@@ -0,0 +1,31 @@
+# Problema 1049 - Beecrowd - Iniciante - Nível 3
+
+# Vamos receber primeiro as três strings, antes de classificar
+stringA = input()
+stringB = input()
+stringC = input()
```

Comment:
```suggestion
filo = input()
classe= input()
alimentacao= input()
```
Para deixar mais didático o código